README.md

```diff
@@ -10,278 +10,184 @@ The main use-case of DiffKemp is to compare selected functions and configuration
-*semantics-preserving patterns*. If a semantic difference is still discovered,
-the relevant diffs are reported to the user.
-
-Note that DiffKemp is incomplete it its nature, hence may provide false positive
+> [!WARNING]
+> DiffKemp is incomplete in its nature, hence may provide false positive
 results (i.e. claiming that functions are not equivalent even though they are).
 This especially happens with complex refactorings.
 
 ## Installation
 
-There are two options to install DiffKemp, either build from source or use a
-prepared RPM package for Fedora.
-
-### Install from source
-
-Currently, DiffKemp runs on Linux and needs the following software installed:
-* Clang and LLVM (supported versions are 9, 10, 11, 12, 13, 14, 15)
-* Python 3 with CFFI (package `python3-cffi` in Fedora and Debian)
-* Python packages from `requirements.txt` (run `pip install -r requirements.txt`)
-* CScope (when comparing versions of the Linux kernel)
-* GoogleTest (gtest) for running the C++ tests (can be vendored by using
-  `-DVENDOR_GTEST=On` in the `cmake` command)
-
-Additionally, to build manually, you need to install the following tools:
-* CMake
-* Ninja build system
-
-Build can be done by running:
-
-    mkdir build
-    cd build
-    cmake .. -GNinja -DCMAKE_BUILD_TYPE=Release
-    ninja
-    cd ..
-
-    pip install -e .
-
-The DiffKemp binary is then located in `bin/diffkemp`.
+You can install DiffKemp:
 
-For running the result viewer, it is necessary to have installed:
-* Node.js (>= 14.x)
-* npm (8)
+- By building it manually [from source](docs/installation.md)
+- From a prepared RPM package that can be installed from our
+  [Copr repository](https://copr.fedorainfracloud.org/coprs/viktormalik/diffkemp/):
 
-When building with `cmake`, use `-DBUILD_VIEWER=On` to install necessary
-packages and build an optimized version of the result viewer.
-
-### Install from RPM
-
-Alternatively, you can use a prepared RPM package for Fedora that can be
-installed from our Copr repository:
-[https://copr.fedorainfracloud.org/coprs/viktormalik/diffkemp/](https://copr.fedorainfracloud.org/coprs/viktormalik/diffkemp/)
+  ```sh
+  # Enabling the DiffKemp repository
+  dnf install -y dnf-plugins-core
+  dnf copr enable -y viktormalik/diffkemp
+  # Installing DiffKemp
+  dnf install -y diffkemp
+  ```
 
 ## Usage
-DiffKemp runs in two phases:
 
-- **Snapshot generation** compiles the compared project versions into LLVM IR
-  and creates so-called *snapshots* which contain the relevant LLVM IR files and
-  additional metadata.
+```mermaid
+---
+config:
+  themeCSS: "
+% redefinition of subgraphs labels
+.cluster-label {
+  font-family: monospace;
+  font-size: 13px;
+}
+"
+  themeVariables:
+    nodeBorder: black
+    mainBkg: white
+    clusterBkg: white
+    clusterBorder: white
+---
+flowchart LR
+    %% nodes
+    pv(project versions)
+    sl(symbol list)
+    subgraph sgSubgraph[diffkemp build\ndiffkemp build-kernel\ndiffkemp llvm-to-snapshot]
```

Comment:
Is there a way to make these labels in "code-style font", i.e. `diffkemp build`, ...? It would help to understand that they are commands.

Reply:
I changed the font to [monospace](https://github.com/PLukas2018/diffkemp/blob/4ae8ff05360d3fa9d353bdef93f589a316d9d082/README.md?plain=1#L41), it looks like it was not previously aplicated because of comment which I added, now it seems to be fine.